perf/src/main.rs

```diff
@@ -0,0 +1,118 @@
+use std::net::{IpAddr, Ipv6Addr, SocketAddr};
+
+use anyhow::{Context, Result};
+use futures::StreamExt;
+use tracing::{debug, error, info};
+
+#[tokio::main]
+async fn main() {
+    tracing_subscriber::fmt::init();
+
+    let cert = rcgen::generate_simple_self_signed(vec!["localhost".into()]).unwrap();
+    let key = quinn::PrivateKey::from_der(&cert.serialize_private_key_der()).unwrap();
+    let cert = quinn::Certificate::from_der(&cert.serialize_der().unwrap()).unwrap();
+
+    let mut server_config = quinn::ServerConfigBuilder::default();
+    server_config
+        .certificate(quinn::CertificateChain::from_certs(vec![cert.clone()]), key)
+        .unwrap();
+    server_config.protocols(&[b"perf"]);
+
+    let server_config = server_config.build();
```

Comment:
I guess this is ok for starting the thing. However I think this should later adjust settings for increasing perf. E.g. flow control limits might be raised. Limits should probably equal to what `bulk` does.